.mailmap

```diff
@@ -105,23 +114,26 @@ Jasper Hugunin <jasper@hugunin.net>                Jasper Hugunin <jasperh@cs.wa
 Jasper Hugunin <jasper@hugunin.net>                Jasper Hugunin <jasper@hashplex.com>
 Tom Hutchinson <thutchin@gforge>                   thutchin <thutchin@85f007b7-540e-0410-9357-904b9bb8a0f7>
 Bodo Igler <bodo.igler@hs-rm.de>                   igler <bodo.igler@hs-rm.de>
-
 Jacques-Henri Jourdan <jacques-henri.jourdan@normalesup.org> Jacques-Henri Jourdan <jacques-henri.jourdan@inria.fr>
 Pierre Jouvelot <pierre.jouvelot@mines-paristech.fr> jouvelot <pierre.jouvelot@mines-paristech.fr>
+Ralf Jung                                          RalfJung
```

Comment:
Does this actually work? (And same thing with Karl below.)

Reply:
Not really :) I'm using the file from the list-contributors script to translate from github pseudos to names. I'll add < > to make it a proper .mailmap file.

Reply:
I fixed the list-contributors script as well to use the new logic, for future release notes.